src/plugins/base64/CMakeLists.txt

```diff
@@ -5,8 +5,6 @@ add_plugin (base64
 		    base64_functions.c
 		    base64.h
 		    base64.c
-	    COMPILE_DEFINITIONS ELEKTRA_PLUGIN_NAME=\"base64\"
```

Comment:
Great to see this simplification!

Similar changes can be done for gpgme, resolver, mmap* (@mpranj)